pex/bin/pex.py

```diff
@@ -289,6 +291,25 @@ def configure_clp_pex_environment(parser):
            'can be passed multiple times to create a multi-interpreter compatible pex. '
            'Default: Use current interpreter.')
 
+  group.add_option(
+    '--interpreter-constraint',
+    dest='interpreter_constraint',
```

Comment:
IIRC, a-b-c will be automatically convert to a_b_c for variable names, so you don't necessarily need to specify dest here.